keyboards/kprepublic/bm40hsrgb/keymaps/dan/rules.mk

```diff
@@ -0,0 +1,8 @@
+# RULES
```

Comment:
This file can be completely deleted, there are no changes vs. the keyboard-level rules.mk.

Reply:
I have deleted the rules.mk file.